WooCommerce/WooCommerceUITests/Mocks/mappings/jetpack-blogs/wc/me/users_me.json

```
@@ -0,0 +1,33 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPath": "/rest/v1.1/jetpack-blogs/161477129/rest-api/",
+    "queryParameters": {
+      "context": {
+        "equalTo": "edit"
+      },
+      "path": {
+        "equalTo": "/wp/v2/users/me&_method=get"
+      },
+      "fields": {
+        "equalTo": "id,username,id_wpcom,email,first_name,last_name,nickname,roles"
```

I believe the UI tests fail on CI because of unmatched WireMock request. After the change below, the UI tests start passing for me locally. The request in question is:

`/rest/v1.1/jetpack-blogs/161477129/rest-api/?json=true&path=/wp/v2/users/me&_method=get&query={"context":"edit","fields":"id,username,id_wpcom,email,first_name,last_name,nickname,roles"}`

- `context` is actually placed inside `query`, while in the mock file it was specified separately
- The mock file contained `fields` as the request matcher value, but it should be `query`
- I saw `context` being physically placed both before and after `fields` in `query`. Unfortunately, WireMock provides no convenient support for `OR` operation, so I thought of removing it completely:

```suggestion
      "path": {
        "equalTo": "/wp/v2/users/me&_method=get"
      },
      "query": {
        "matches": "(.*)\"fields\":\"id,username,id_wpcom,email,first_name,last_name,nickname,roles\"(.*)"
```